src/uwtools/config/support.py

```diff
@@ -66,17 +68,19 @@ def log_and_error(msg: str) -> Exception:
 
 
 # Private functions
+
+
 def _represent_namelist(dumper: yaml.Dumper, data: Namelist) -> yaml.nodes.MappingNode:
     """
     Convert f90nml Namelist to OrderedDict and serialize.
 
     :param dumper: The YAML dumper.
     :param data: The f90nml Namelist to serialize.
     """
-    # Convert the f90nml Namelist to an OrderedDict.
-    namelist_dict = data.todict()
 
-    # Represent the OrderedDict as a YAML mapping.
+    # Convert the f90nml Namelist to an OrderedDict, then represent as a YAML mapping.
```

Comment:
I'm curious if this could be the docstring info, and we don't need an additional comment since it's only a 2-liner.

Reply:
Done.